GCEm/abc_sampler.py

```diff
@@ -152,6 +154,8 @@ def constrain(implausibility, tolerance=0., threshold=3.0):
     threshold = tf.constant(threshold, dtype=implausibility.dtype)
 
     # Count the number of implausible observations against the threshold
+    #  Note that the tf.greater comparison will return False for any NaN implausibilities,
+    #  hence NaN observations are always plausible.
```

Comment:
I don't understand this stuff as well as you/lucia, but isn't this saying that if the observation is `NaN` then the model can predict anything and not be penalized for it?

Reply:
/Does that make sense?

Reply:
Yes, that's exactly right. I *think* it's the only thing that makes sense since (by definition) we can't compare with NaNs...